### DISCUSSION TITLE
Call to undefined method YellowCore::getRequestHandler in private extension

### DISCUSSION THREAD

**Discussion by IngoBelka** [datenstrom/community#867] [Report a bug] [answered]
I did installed "Private" extension. Then I created a new private page. Now I get an error message, if I want to go to this page:

`Fatal error: Uncaught Error: Call to undefined method YellowCore::getRequestHandler() in /var/www/html/kochschule/system/extensions/private.php:20 Stack trace: #0 /var/www/html/kochschule/system/extensions/core.php(3151): YellowPrivate->onParsePageLayout() #1 /var/www/html/kochschule/system/extensions/core.php(3111): YellowPage->parsePageLayout() #2 /var/www/html/kochschule/system/extensions/core.php(195): YellowPage->parsePage() #3 /var/www/html/kochschule/system/extensions/core.php(141): YellowCore->sendPage() #4 /var/www/html/kochschule/system/extensions/edit.php(297): YellowCore->processRequest() #5 /var/www/html/kochschule/system/extensions/edit.php(259): YellowEdit->processRequestShow() datenstrom/community#7 /var/www/html/kochschule/system/extensions/edit.php(68): YellowEdit->processRequest() datenstrom/community#8 /var/www/html/kochschule/system/extensions/core.php(106): YellowEdit->onRequest() #8 /var/www/html/kochschule/yellow.php(9): YellowCore->request() datenstrom/community#9 {main} thrown in /var/www/html/kochschule/system/extensions/private.php on line 20`

I'm not familiar enough with programming, to understand this debug message.

**Comment by schulle4u** [comment, marked as answer]
That's why we are here, so thanks for the bug report. 🙂 Please re-download and install the extension again, this should fix the error. Apparently I missed some API changes.

**Comment by IngoBelka** [comment]
Fast and easy. :) Thank you Steffen!
